CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_zendesk v0.20.0
+
+## [Upstream Under-the-Hood Updates](https://github.com/fivetran/dbt_zendesk_source/blob/main/CHANGELOG.md) from `zendesk_source` Package
```

Comment:
Upstream and "from `zendesk_source` Package is redundant. I'd suggest keeping one or the other.

I'd link directly to the release notes for the source package rather than the CHANGELOG, as the CHANGELOG will change over time.  Either that or the release hyperlink (https://github.com/fivetran/dbt_zendesk/blob/main/CHANGELOG.md#dbt_zendesk_source-v0200) I think will be the link? Might want to check.

Reply:
thanks! updated